lms/djangoapps/teams/toggles.py

```diff
@@ -0,0 +1,38 @@
+"""
+Togglable settings for Teams behavior
+"""
+from edx_toggles.toggles import SettingDictToggle
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+
+
+# Course Waffle inherited from edx/edx-ora2
+WAFFLE_NAMESPACE = "openresponseassessment"
+TEAM_SUBMISSIONS_FLAG = "team_submissions"
+
+# .. toggle_name: FEATURES['ENABLE_ORA_TEAM_SUBMISSIONS']
+# .. toggle_implementation: SettingDictToggle
+# .. toggle_default: False
+# .. toggle_description: Set to True to enable team-based ORA submissions.
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2020-03-03
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: https://openedx.atlassian.net/browse/EDUCATOR-4951
+# .. toggle_warnings: This temporary feature toggle does not have a target removal date.
+TEAM_SUBMISSIONS_FEATURE = SettingDictToggle(
+    "FEATURES", "ENABLE_ORA_TEAM_SUBMISSIONS", default=False, module_name=__name__
+)
+
+
+def are_team_submissions_enabled(course_key):
```

Comment:
1. I know file renames are imperfect in git, but not sure if this was handled like a rename or not and if github just isn't picking it up?
2. Moving code adds risk, because plugins, etc. might use existing calls. Reminder to search all edx repos, which you may have done.  This looks ok.

Reply:
> Moving code adds risk, because plugins, etc. might use existing calls. Reminder to search all edx repos, which you may have done. This looks ok.

Yes, this looks fine: https://github.com/search?q=are_team_submissions_enabled&type=code